lib/Updater.php

```diff
@@ -223,7 +223,6 @@ private function getExpectedElementsList(): array {
 			'lib',
 			'ocs',
 			'ocs-provider',
-			'ocm-provider',
```

Comment:
Maybe it could be added to list if current Nextcloud version is lower than 28?

Reply:
This updater runs on 28 only though. Others are in the stable branches.